__tests__/verifyMocks.test.js

```diff
@@ -39,4 +43,16 @@ describe('Verify that the mocks are unchanged', () => {
       expect(styleDictFigma).toMatchSnapshot();
     });
   });
+
+  describe('for Adobe XD', () => {
+    it.skip('-> Raw dump', () => {
```

Comment:
Figma and Sketch also contain a snapshot test for the sample dump (also skipped). Can we not remove them?